### PR TITLE
Add CHANGELOG for https://github.com/rails/rails/pull/23734 [ci skip]

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
r? @schneems

### CODE REVIEW

**Comment by rafaelfranca** [comment]
This is a change in the generated templates, so it is not

> Add ability to log to STDOUT in production using environment variable `RAILS_LOG_TO_STDOUT`.

It is more like:

> Generating application with a option to log to STDOUT using the environment variable `RAILS_LOG_TO_STDOUT`.

File: railties/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
     *claudiob*
 
+*   Generate applications with an option to log to STDOUT in production
+    using the environment variable `RAILS_LOG_TO_STDOUT`.
+
+    *Richard Schneeman*
+
```
Comment (prathamesh-sonpatki) [inline railties/CHANGELOG.md]: Keeping it as second entry so as to honour actual order of commits.

**Comment by prathamesh-sonpatki** [comment]
@rafaelfranca Done, makes sense.

**Comment by rafaelfranca** [comment]
Thank you  :heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**